vm/gvisor/gvisor.go

```diff
@@ -104,8 +104,28 @@ func (pool *Pool) Create(workdir string, index int) (vmimpl.Instance, error) {
 		return nil, err
 	}
 
+	panicLog := filepath.Join(bundleDir, "panic.fifo")
+	if err := syscall.Mkfifo(panicLog, 0666); err != nil {
```

Comment:
Does this need to be a fifo? Or a pipe will do?
A pipe can be created much simpler with `osutil.LongPipe`.

Reply:
We need to specify a file path for the --panic-log option. Do you suggest to specify a path to /proc/PID/fd/X? I tried this, but this doesn't work due to permission checks.

Reply:
Ah, I see, I missed that part.